### PR TITLE
[DOCS] How-to guide for validation without a checkpoint

### PR DESCRIPTION
**Pull request by spbail**
Adding content for the stub. Listing out the different types of batch_kwargs might be controversial and go against our principle of keeping the how-to guides minimal. Feel free to suggest other good places to put these.

### CODE REVIEW

File: docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst [hunk 1]
```diff
@@ -3,16 +3,58 @@
 How to validate data without a Checkpoint
 =========================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide demonstrates how to load an Expectation Suite and validate data without using a :ref:`Checkpoint<how_to_guides__validation__how_to_create_a_new_checkpoint>`. This might be suitable for environments or workflows where a user does not want to or cannot create a Checkpoint.
```
Comment (abegong) [inline docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst]: This begs the question "When should I use a Checkpoint vs using some other workflow?"

Is there a simple way to answer this in one sentence, or should we write a separate article to cover it?

File: docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst [hunk 2]
```diff
@@ -3,16 +3,58 @@
...
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
 
+  - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+  - :ref:`Created an Expectation Suite <how_to_guides__creating_and_editing_expectations>`
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+The following code mirrors the code provided in the ``validation_playground.ipynb`` notebooks in ``great_expectations/notebooks``. First of all, we import Great Expectations, load our :ref:`Data Context<data_context>`, and define variables for the Datasource and Expectation Suite we want to access:
```
Comment (abegong) [inline docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst]:
```suggestion
The following code mirrors the code provided in the ``validation_playground.ipynb`` notebooks in ``great_expectations/notebooks``. First of all, we import Great Expectations, load our :ref:`Data Context<data_context>`, and define variables for the Datasource we want to access:
```

We don't actually do the Exp Suite until much later, so I wouldn't mention it yet.

File: docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst [hunk 3]
```diff
@@ -3,16 +3,58 @@
...
+
+    batch = context.get_batch(batch_kwargs, "my_expectation_suite")
+
+    results = context.run_validation_operator(
+        "action_list_operator",
+        assets_to_validate=[batch],
+        run_id="my_run_id") # Make my_run_id a unique identifier, e.g. a timestamp
+
+
+This runs validation and saves the results to JSON.
```
Comment (abegong) [inline docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst]:
```suggestion
This runs validation and executes any ValidationActions configured for this ValidationOperator (e.g. saving the results to a ValidationResult Store).
```

It won't always be JSON...

**Review by abegong — CHANGES_REQUESTED** [review]
This is great---three changes I think are worth making.

The first is the most important, IMO. We need to speak with one voice on this.

Also, the whole middle part about batch_kwargs makes my heart ache. I can't wait to dispose of that nonsense after 0.13.0.

File: docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst (hunk 1 above)
Comment (spbail) [reply docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst]: I'd leave explanation out of the how-tos,  but I can say "for example in a hosted environment"?

File: docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst (hunk 2 above)
Comment (spbail) [reply docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst]: Good catch!

File: docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst (hunk 3 above)
Comment (spbail) [reply docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst]: Good call, thanks!

**Review by spbail — COMMENTED** [review]
Made the fixes!

File: docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst (hunk 1 above)
Comment (abegong) [reply docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst]: Sure, that makes sense.

**Review by abegong — APPROVED** [review]

**Pull request merged**